### PR TITLE
TF Keras io_utils_test missing test cases add

### PR DESCRIPTION
**Pull request by Dayananda-V**
1-test case added for ask_to_proceed_with_overwrite

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewers @fchollet, @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewers @fchollet, @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Review by tomerk — APPROVED** [review]
